src/content/upgrades/merge/issuance/index.md

```diff
@@ -14,88 +18,116 @@ The **issuance** of ETH is the process of creating ETH that did not previously e
+- Using 14,000,000 total ETH staked, the rate of ETH issuance is approximately 1700 ETH/day ([See source](https://ultrasound.money/))
+- Results in **~620,500** ETH issued in a year
+- Resulted in inflation rate of **approximately 0.52%** (620.5K per year / 119.3M total)
 
 <InfoBanner>
-<strong>Total annual issuance rate: ~4.62%</strong> (4.13% + 0.49%)<br/><br/>
-<strong>~89.4%</strong> of the issuance is going to miners on the execution layer (4.13 / 4.62 * 100)<br/><br/>
-<strong>~10.6%</strong> is being issued to stakers on the consensus layer (0.49 / 4.62 * 100)
+<strong>Total annualized issuance rate (pre-merge): ~4.61%</strong> (4.09% + 0.52%)<br/><br/>
+<strong>~88.7%</strong> of the issuance was going to miners on the execution layer (4.09 / 4.61 * 100)<br/><br/>
+<strong>~11.3%</strong> was being issued to stakers on the consensus layer (0.52 / 4.61 * 100)
 </InfoBanner>
 
-## Post-merge {#post-merge}
+## Post-merge (present day) {#post-merge}
 
 ### Execution layer issuance {#el-issuance-post-merge}
 
-Execution layer issuance after The Merge will be zero. Proof-of-work will no longer be valid under the rules of consensus. All execution layer activity will be included in "beacon blocks", which are published and attested to by proof-of-stake validators.
+Execution layer issuance since The Merge is zero. Proof-of-work is no longer a valid means of block production under the upgraded rules of consensus. All execution layer activity is packaged into "beacon blocks", which are published and attested to by proof-of-stake validators. Rewards for attesting-to and publishing beacon blocks are accounted for separately on the consensus layer.
 
 ### Consensus layer issuance {#cl-issuance-post-merge}
 
-Consensus layer issuance will continue as before The Merge, with small rewards for validators who attest to and propose blocks. Validator rewards will continue to accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.
+Consensus layers issuance continues today as it did prior to The Merge, with small rewards for validators who attest to and propose blocks. Validator rewards continue to accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the upcoming Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% remains locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.
```

Comment:
```suggestion
Consensus layer issuance continues today as it did prior to The Merge, with small rewards for validators who attest to and propose blocks. Validator rewards continue to accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the upcoming Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% remains locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.
```

Think this reads a little more clear, but not a string opinion

Reply:
Also noting the text around shanghai upgrade. Im fine with this as Vitalik and others have basically said it will be in Shanghai, but we did have comments in the past about this

Reply:
Yeah, despite the exact list of EIPs not being solidified yet, there seems to be consensus on withdraws being prioritized and I'm comfortable with the wording at this point.